contracts/dependencies/swap/IUniswapV2LikeRouter.sol

```diff
@@ -5,25 +5,55 @@ interface IUniswapV2LikeRouter {
     function swapExactTokensForTokens(
         uint256 amountIn,
         uint256 amountOutMin,
-        address[] memory path,
+        address[] calldata path,
         address to,
         uint256 deadline
     ) external returns (uint256[] memory amounts);
 
     function swapTokensForExactTokens(
         uint256 amountOut,
         uint256 amountInMax,
-        address[] memory path,
```

Comment:
We are using those interfaces to call external contracts, and even whether it is declared memory or calldata, solidity compiler will encode the 'path' to calldata format. Pretty much all the projects are checked are using 'calldata' here. (even us in our own Brokkr interfaces)

@jihoonsong let's talk about this offline if you don't agree